Makefile

```diff
@@ -356,6 +364,9 @@ $(GOLANGCI_LINT): # Download and install golangci-lint
 		-b $(TOOLS_BIN_DIR) \
 		$(GOLANGCI_LINT_VER)
 
+$(NOTES): # Download and install note generator from cluster-api commit
```

Comment:
we need to remove capi notes, perhaps?

Reply:
None of the created binaries are getting removed by any of the makefile tasks, why should it be?

Reply:
@furkatgofurov7 There is no nice way to import CAPI release tool here, all methods are private. The only option is to build it from CAPI repo. We can talk with CAPI folks about converting it to a github action.